libraries/ae-security/src/main/java/org/metaeffekt/core/security/cvss/CvssSeverityRanges.java

```diff
@@ -85,10 +85,24 @@ public SeverityRange getRangeByName(String name) {
         return UNDEFINED_SEVERITY_RANGE;
     }
 
+    //TODO: consider replace for 'getRangeByName'
```

Comment:
TODO (on this PR) or FIXME (for any time later). When FIXME then please insert a space: 
`// FIXME: consider replace for 'getRangeByName'`